api/tests/integration/infrastructure/repositories/attachable-target-profiles-repository_test.js

```diff
@@ -90,6 +90,28 @@ describe('Integration | Repository | attachable-target-profiles', function () {
         // then
         expect(results).to.deep.equal([{ id: 100, name: 'currentlyDetached' }]);
       });
+
+      context('when the target profile has an history where it has been detached', function () {
+        it('should not return a target profile even if it was detached in the past', async function () {
```

Comment:
Chipotage: le contexte est déjà présent ligne 94

```suggestion
        it('should not be returned', async function () {

Reply:
Done